district_data_import.py

```diff
@@ -154,7 +154,7 @@
 
         # Update address
         dp_donorrecord.update({
-            'ADDRESS': district_record['Street'],
+            'ADDRESS': district_record['Mailing_Street'],
             'CITY': district_record['City'],
```

Comment:
Shouldn't we be using mailing_city, mailing_state, and mailing_zip?  If we're using the mailing address, we should use all parts as they may have a po box for mailing and it could be in another city.  I've seen data that has different address/city for regular address and mailing address.

Reply:
Yes, that makes sense. I've updated both PRs.